app/src/main/java/org/jellyfin/androidtv/preferences/UserPreferences.kt

```diff
@@ -170,5 +165,14 @@ class UserPreferences(context: Context) : SharedPreferenceStore(PreferenceManage
 			// Migrate to login behavior enum
 			putEnum("login_behavior", if (it.getString("pref_login_behavior", "0") == "1") LoginBehavior.AUTO_LOGIN else LoginBehavior.SHOW_LOGIN)
 		}
+
+		migration(toVersion = 4) {
```

Comment:
I think we should add this new enum to the previous migration (`toVersion = 3`). My intention was to have 1 migration per app release. _The initial update will have 2 though, 1 for the setting change in 0.11 for people that upgrade from 0.10->0.12_

Reply:
:+1: Cool sounds good to me. I had wondered about that.